helpers/jwt-service.go

```diff
@@ -0,0 +1,40 @@
+package helpers
+
+import (
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/joho/godotenv"
+)
+
+type JWTService interface {
+	GenerateToken(user_id string) (string, error)
+}
+
+type JWTServiceInfo struct {
+	secretKey string
+	issuer    string
+}
+
+func NewJWTService() JWTService {
+	envError := godotenv.Load()
+	AlertError(envError, "The .env file could not be found")
+	JWTSecret := os.Getenv("JWT_SECRET")
+	return &JWTServiceInfo{
+		secretKey: JWTSecret,
+		issuer:    "drift",
```

Comment:
Seems like this was copied from the gitlab example, but you're not actually using it in `jwt.StandardClaims{`. Not sure if you wanted to use it? Otherwise remove.

Reply:
Addressed above